builtin/game/voxelarea.lua

```diff
@@ -48,20 +48,19 @@ function VoxelArea:indexp(p)
 end
 
 function VoxelArea:position(i)
-	local p = {}
 	local MinEdge = self.MinEdge
 
 	i = i - 1
 
-	p.z = math.floor(i / self.zstride) + MinEdge.z
+	local z = math.floor(i / self.zstride) + MinEdge.z
 	i = i % self.zstride
 
-	p.y = math.floor(i / self.ystride) + MinEdge.y
+	local y = math.floor(i / self.ystride) + MinEdge.y
 	i = i % self.ystride
 
-	p.x = math.floor(i) + MinEdge.x
+	local x = math.floor(i) + MinEdge.x
 
-	return p
+	return vector.new(x, y, z)
```

Comment:
Consider localizing `math.floor` and `vector.new` as this is a performance-critical function.

Reply:
Is this OK for readability?
```lua
local math_floor = math.floor
local vector_new = vector.new
```

Reply:
> Is this OK for readability?
> 
> ```lua
> local math_floor = math.floor
> local vector_new = vector.new
> ```

Yes, absolutely.

Reply:
Fixed